addons/skin.estuary/xml/Includes.xml

```diff
@@ -859,7 +862,7 @@
 					<animation effect="fade" time="150">VisibleChange</animation>
 					<control type="group">
 						<width>600</width>
-						<visible>Player.HasMedia + String.IsEmpty(Window(Videos).Property(PlayingBackgroundMedia))</visible>
+						<visible>!Control.IsVisible(503) + Player.HasMedia + String.IsEmpty(Window(Videos).Property(PlayingBackgroundMedia))</visible>
 						<animation effect="fade" time="200">VisibleChange</animation>
```

Comment:
This removes the Now Playing info a the top when the Now Playing view is active

**Widelist view with now playing info still at top**

![image](https://user-images.githubusercontent.com/5781142/70863267-0e812700-1f3e-11ea-9b08-5226e373517f.png)

**Now Playing view with now playing info at top removed**

![image](https://user-images.githubusercontent.com/5781142/70863275-353f5d80-1f3e-11ea-9d2a-04364213fd5c.png)

Due this view providing the now playing info in the right hand panel this info at the stops is redundant while in this view, however I don't have a strong opinion on this either way.